vyper/ast/nodes.py

```diff
@@ -738,6 +748,16 @@ class Bytes(Constant):
     __slots__ = ()
     _translated_fields = {"s": "value"}
 
+    def __init__(self, parent: Optional["VyperNode"] = None, **kwargs: dict):
+        super().__init__(parent, **kwargs)
+        if isinstance(self.value, str):
+            self.value = self.value.encode("utf8")
+
+    def to_dict(self):
+        ast_dict = super().to_dict()
+        ast_dict["value"] = self.value.decode("utf8")
```

Comment:
would it be better to convert to hex?

Reply:
I think string is better. It's simpler, it more accurately preserves the original value, and using hex would make for some weird ambiguity between the `Bytes` and `Hex` nodes.